src/excel/utilities.ts

```diff
@@ -1,31 +1,30 @@
 /* Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. */
 
-import { CustomError } from '../errors/custom.error';
+import { APIError } from '../errors/custom.error';
 
 /**
- * Custom error type to handle API specific errors.
- */
-export class APIError extends CustomError {
-    /**
-     * @constructor
-     *
-     * @param message Error message to be propagated.
-     * @param state OAuth state if available.
-    */
-    constructor(message: string, public innerError?: Error) {
-        super('APIError', message, innerError);
-    }
-}
-
-/**
- * Helper exposing useful Utilities for Excel-Addins.
+ * Helper exposing useful Utilities for Excel Add-ins.
  */
 export class ExcelUtilities {
     /**
-     * Utility to delete a worksheet if it already exists.
-     * @returns true if the sheet had existed before being deleted.
+     * Utility to create (or re-create) a worksheet, even if it already exists.
+     * @returns the new worksheet
      */
-    static async forceCreateSheet(workbook: Excel.Workbook, sheetName: string): Promise<Excel.Worksheet> {
+    static async forceCreateSheet(
+        workbook: Excel.Workbook,
+        sheetName: string,
+        options?: {
```

Comment:
options bag mostly for the self-documenting nature of the boolean.

As you might imagine, the case for this particular modification (and the original method) came from a very concrete need in a snippet.

And to me, something like:

            const sheet = await forceCreateSheet(context.workbook, "Sample", { clearOnly: true });

Is a lot simpler to reason about than

            const sheet = await forceCreateSheet(context.workbook, "Sample", true);

So even if we may never expose other options, I think that booleans especially -- unless it's a method like `setVisible`, where it's very obvious what the `true` or `false` values would imply -- are better in property bags.  And I can (maybe) imagine a world in which we want another option someday (e.g., `addSuffixIfExists`), to make it give you `Sample1`, `Sample2`, etc.